linters/jshint.py

```diff
@@ -0,0 +1,13 @@
+def options(ctx):
+    ctx.add_option('--jshint', action='store_true', help="Run JSHint on the JS files in the build")
+
+def configure(ctx):
+    if ctx.options.jshint:
+        try:
+            ctx.find_program('jshint', var='JSHINT')
+        except ctx.errors.ConfigurationError:
+            print "jshint was not found"
+
+def build(ctx):
+    if ctx.env.JSHINT:
+        ctx(rule='${JSHINT} ${SRC}', source=ctx.path.ant_glob('src/**/*.js'), name='jshint')
```

Comment:
There are a lot of things I dislike about this line, but I can't think of any significantly better alternative. My only true objection is that there are no constraints on which order this will run relative to other tasks, so the pbz could be built before jshint has a chance to fail the build. But this rule would have to be put in a feature in order to control build order.